common/app/model/content/Atoms.scala

```diff
@@ -53,6 +54,16 @@ object MediaAssetPlatform extends Enum[MediaAssetPlatform] with PlayJsonEnum[Med
   case object Url extends MediaAssetPlatform
 }
 
+sealed trait MediaWrapper extends EnumEntry
+
+object MediaWrapper extends Enum[MediaWrapper] with PlayJsonEnum[MediaWrapper] {
+  val values = findValues
+
+  case object MainMedia extends MediaWrapper
+  case object ImmersiveMainMedia extends MediaWrapper
+  case object EmbedPage extends MediaWrapper
```

Comment:
Would it make sense to have an additional `None` value (ex: NoMediaWrapper). So you don't have to carry Option[MediaWrapper] all over the place, as well as simplifying comparison?